### ISSUE TITLE
Trait-based quests

### ISSUE THREAD

**Issue by tukkek** [tukkek/javelin#248]
Originally from #247 

> There could also be trait-related quests - for example, Magic cities could ask for low/mid/high/epic-level magic items (of a particular type). A systemic way to approach this would be to have monsters drop "trophies", where each type of town would want as quests - magic towns might want aberration trophies for research and military towns might want Dragon trophies for glory. The fact that these would not be easily focus-farmed right now is an issue though.

**Comment by tukkek** [comment]
Towns should also give out eventual Ruby rewards from Treaties. After #288, I made it so that the price to recruit an unit is its CR-worth of rubies, so even getting all the Dungeon rubies you'd only find something around the recruitment of a single CR20 creature in the entire world map...

... which is fine, it should be perfectly possible to recruit lower-level units and build them up but players shouldn't be forced to dungeon-crawl if they don't enjoy that playstyle so Quests providing Rubies is still important for the sandbox aspect of it.

**Comment by tukkek** [comment]
Closing this in favor of #247.
